test/integration/suite.spec.js

```diff
@@ -13,7 +13,8 @@ describe('suite w/no callback', function() {
         if (err) {
           return done(err);
         }
-        var result = res.output.match(/no callback was supplied/) || [];
+        var pattern = /MissingCallbackError/g;
```

Comment:
You can use `expect` here; `assert` is just leftover and should be replaced anyway